libs/utils/include/utils/android/ThermalManager.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef TNT_UTILS_ANDROID_THERMALMANAGER_H
+#define TNT_UTILS_ANDROID_THERMALMANAGER_H
+
+#include <utils/compiler.h>
+
+struct AThermalManager;
+
+namespace utils {
+
+class ThermalManager {
+public:
+    enum class ThermalStatus : int8_t {
+        ERROR = -1,
```

Comment:
Could the enum be shared between both headers?

Reply:
We should probably revisit how we handle platform specific implementations in `libutils` but the way it's done currently, is by duplicating the headers  and including the right one at compile time. Sharing the types/enums between them makes things complicated, e.g. introducing a new header that's public but not to be included by the user.